contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -191,7 +191,10 @@ contract InitializableAbstractStrategy is Initializable, Governable {
         public
         onlyGovernor
     {
-        IERC20(_asset).transfer(governor(), _amount);
+        require(
+            IERC20(_asset).transfer(governor(), _amount),
```

Comment:
Need to merge https://github.com/OriginProtocol/origin-dollar/pull/378 before this as it no longer applies

Reply:
Have merged master with this branch and reverted these back.